during_workshop.md

```diff
@@ -87,55 +87,17 @@ cd AcmeSubProject
 Next, we want to create a function that will be triggered by an HTTP request, so we will use the 'HTTP trigger' template.
 
 ```
-func new --name HttpEndpoint --template "HTTP trigger" --authlevel "anonymous"
+func new --name AddSubtitle --template "HTTP trigger"
 ```
+(when prompted choose the ANONYMOUS option for the Auth Level)
 
 This creates a new subfolder, containing the code for this function:

```

Comment:
For me this didn't create a _new_ subfolder like it used to, it just amended the file that had been created by the previous command?

Reply:
Ah that's absolutely correct, good spot!